src/zc/buildout/easy_install.py

```diff
@@ -1322,6 +1323,11 @@ def _pyscript(path, dest, rsetup, initialization=''):
 
 if _interactive:
     del _interactive
+
+    filename = os.environ.get('PYTHONSTARTUP')
+    if filename and os.path.isfile(filename):
+        exec(open(filename).read())
+
```

Comment:
I think you should use the `with` statement here, and probably open the file in 'U' mode like the branch above.  Also, an explicit compile() step, again like in the above code (lines 1321-1322), so syntax errors get the right filename.

I also think the `filename` global should be renamed to start with an underscore, to blend with the code around it.  These names are global and leak into the interactive session, unless you take care to `del` them, which I think you should also do.